sql_generators/mobile_kpi_support_metrics/templates/activation_clients.query.sql

```diff
@@ -0,0 +1,82 @@
+WITH new_profiles AS (
+  SELECT
+    first_seen_date,
+    client_id,
+    normalized_channel,
+    app_name,
+    app_version,
+    country,
+    city,
+    geo_subdivision,
+    locale,
+    isp,
+    os,
+    os_version,
+    device_model,
+    device_manufacturer,
+    is_mobile,
+    {% for attribution_field in product_attribution_fields %}
+    {{ attribution_field }},
+    {% endfor %}
+    device_type,
+  FROM
+    `{{ project_id }}.{{ dataset }}.new_profile_clients`
+  WHERE
+    first_seen_date = DATE_SUB(@submission_date, INTERVAL 6 DAY)
+),
+active_users AS (
+  SELECT
+    first_seen_date,
+    client_id,
+    ARRAY_LENGTH(
+      mozfun.bits28.to_dates(mozfun.bits28.range(days_seen_bits, -5, 6), submission_date)
```

Comment:
good opportunity to fix the definition and use active instead of seen

Reply:
Added it, calling it `is_early_engagement` as agreed in addition is `is_activated`